### ISSUE TITLE
Crash on startup migrating database

### ISSUE THREAD

**Issue by mbenke** [golemfactory/clay#4679]
## Description

**Golem Version**:  0.20.1+dev551.gebf7ef5
**Golem-Messages version** (leave empty if unsure): 3.11.0

**Electron version** (if used): N/A

**OS** [e.g. Windows 10 Pro]: Ubuntu 16.04

**Branch** (if launched from source): develop (commit: ebf7ef5135f268a074dd7f32daad401ca834885b)

**Mainnet/Testnet**: testnet

**Description of the issue**:

Crash on startup (Migrating database schema)

**Actual result**:

No crash

**Screenshots**:

N/A

## Steps To Reproduce

run golemapp with datadir containing version 34 of the database

## Logs and any additional context

```
INFO     [golemapp                           ] GOLEM Version: 0.20.1+dev551.gebf7ef5
INFO     [golemapp                           ] Protocol Version: 64022-testnet
INFO     [golemapp                           ] golem_messages Version: 3.11.0
INFO     [golemapp                           ] system: Linux, release: 4.15.0-55-generic, version: #60~16.04.2-Ubuntu SMP Thu Jul 4 09:03:09 UTC 2019, machine: x86_64
INFO     [golemapp                           ] cpu: GenuineIntel Intel(R) Core(TM) i5-4690 CPU @ 3.50GHz, 4 cores
INFO     [golemapp                           ] memory: 15.5 GiB, swap: 14.0 GiB
INFO     [golemapp                           ] Ethereum chain: rinkeby
INFO     [golemapp                           ] Concent url: https://test.concent.golem.network
INFO     [golemapp                           ] Concent public key: f008d9805609f32742838d54eca7731dfd4392a87d474d940346ea46d805ea6ad970347c79ef0be094b340d27bf69047207f341d0f36cdbaf85e022c3b91dbcd
INFO     [golem.ethereum.web3.providers      ] GETH: connecting to remote RPC interface at http://94.23.57.58:55555

INFO     [golem.db                           ] Migrating database schema from version 34 to 35

WARNING  [golem.db                           ] execute_sql() retry timeout after 112613 iterations. Giving up. sql='SELECT details, status, payee, value, subtask, created_date FROM payment', params=None
no such table: payment
Traceback (most recent call last):
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
sqlite3.OperationalError: no such table: payment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee_migrate/router.py", line 131, in run_one
    migrate(migrator, self.database, fake=fake)
  File "<string>", line 67, in migrate
  File "/home/ben/github/golem/golem/database/database.py", line 30, in execute_sql
    return super().execute_sql(sql, params, require_commit)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3837, in execute_sql
    self.commit()
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3656, in __exit__
    reraise(new_type, new_type(*exc_args), traceback)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 135, in reraise
    raise value.with_traceback(tb)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
peewee.OperationalError: no such table: payment
ERROR    [peewee_migrate                     ] no such table: payment
Traceback (most recent call last):
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
sqlite3.OperationalError: no such table: payment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee_migrate/router.py", line 131, in run_one
    migrate(migrator, self.database, fake=fake)
  File "<string>", line 67, in migrate
  File "/home/ben/github/golem/golem/database/database.py", line 30, in execute_sql
    return super().execute_sql(sql, params, require_commit)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3837, in execute_sql
    self.commit()
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3656, in __exit__
    reraise(new_type, new_type(*exc_args), traceback)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 135, in reraise
    raise value.with_traceback(tb)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
peewee.OperationalError: no such table: payment
Migration failed: 031_migrate_payment_to_task_payment
ERROR    [peewee_migrate                     ] Migration failed: 031_migrate_payment_to_task_payment
Traceback (most recent call last):
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
sqlite3.OperationalError: no such table: payment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "golemapp.py", line 295, in <module>
    main()
  File "golemapp.py", line 243, in main
    start()  # pylint: disable=no-value-for-parameter
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "golemapp.py", line 204, in start
    _start()
  File "golemapp.py", line 187, in _start
    if crossbar_serializer else None),
  File "/home/ben/github/golem/golem/node.py", line 137, in __init__
    db, fields=DB_FIELDS, models=DB_MODELS, db_dir=datadir)
  File "/home/ben/github/golem/golem/database/database.py", line 90, in __init__
    self._migrate_schema(version, to_version=self.SCHEMA_VERSION)
  File "/home/ben/github/golem/golem/database/database.py", line 122, in _migrate_schema
    migrate_dir=self.schemas_dir)
  File "/home/ben/github/golem/golem/database/migration/migrate.py", line 40, in migrate_schema
    router.run_one(script, migrator, fake=True)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee_migrate/router.py", line 131, in run_one
    migrate(migrator, self.database, fake=fake)
  File "<string>", line 67, in migrate
  File "/home/ben/github/golem/golem/database/database.py", line 30, in execute_sql
    return super().execute_sql(sql, params, require_commit)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3837, in execute_sql
    self.commit()
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3656, in __exit__
    reraise(new_type, new_type(*exc_args), traceback)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 135, in reraise
    raise value.with_traceback(tb)
  File "/home/ben/.virtualenvs/fresh/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
peewee.OperationalError: no such table: payment
Sentry is attempting to send 1 pending error messages
```

**Comment by Wiezzel** [comment]
Probably related to #4341

**Comment by etam** [comment]
fixed by #4437
